src/patternfly/components/Alert/alert.scss

```diff
@@ -120,15 +122,23 @@
 .pf-c-alert__icon {
   grid-area: icon;
   display: flex;
+  align-items: center;
```

Comment:
This will center the icon vertically when the title text wraps to more than 1 line.

I tried out removing the `line-height` added to `__icon > *` and `__title`, and adding back `--pf-c-alert__icon--MarginTop: 1px` and that seems to work, except the close button adds extra height to the top row (both in this PR and with the updates I mentioned). Adding `line-height: 1;` to the close button looks to address that.

Also now I wonder if we should add `line-height: 1` to the button component's base styles, and use the `spacer--sm` for the top/bottom padding instead of the custom `6px` form element var.

Reply:
Here are those changes in a PR - https://github.com/christiemolloy/patternfly-next/pull/27

Reply:
Good point I didnt think about the case where the text wraps to two lines. Merged your PR in that looks great

Reply:
Good point I didnt think about the case where the text wraps to two lines. Merged your PR in that looks great going to check with design about the new font-size